src/main/java/de/embl/cba/mobie2/display/SegmentationDisplay.java

```diff
@@ -75,7 +78,50 @@ public boolean showScatterPlot()
 		return tables;
 	}
 
-	public void updateSerializableFields(){
-		// TODO
-	};
+	/**
+	 * Create a serializable copy
+	 *
+	 * @param segmentationDisplay
+	 */
+	public SegmentationDisplay( SegmentationDisplay segmentationDisplay )
+	{
+		this.name = segmentationDisplay.name;
+		this.sources = new ArrayList<>();
+		for ( SourceAndConverter< ? > sourceAndConverter : segmentationDisplay.sourceAndConverters )
+		{
+			sources.add( sourceAndConverter.getSpimSource().getName() );
+		}
+
+		final SourceAndConverter< ? > sourceAndConverter = segmentationDisplay.sourceAndConverters.get( 0 );
+
+		if( sourceAndConverter.getConverter() instanceof AdjustableOpacityColorConverter)
+		{
+			this.opacity = ( ( AdjustableOpacityColorConverter ) sourceAndConverter.getConverter() ).getOpacity();
+		}
+
+		this.lut = segmentationDisplay.coloringModel.getColoringLUTName();
```

Comment:
@K-Meech 
There is for some of the coloring models already a method `getARGBLut()`
e.g. `CategoryTableRowColumnColoringModel` has this.
I think we should go into imagej-utils and add the method `getARGBLut()` to the other coloring models where we have it (e.g. `LazyCategoryColoringModel` `NumericTableRowColumnColoringModel`).
Then we do not need the `getColoringLUTName()`.
Do you want to do this?
I can also accept the PR and do it myself. No problem!

Reply:
Makes sense! If you had time to do this - that'd be great :)